features/admin.claims.v1/components/edit/local-claim/edit-additional-properties-local-claims.tsx

```diff
@@ -17,14 +17,19 @@
  */
 
```

Comment:
Update the license.

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7106/commits/3088d327c477257341387a57516db4d9929fbfaa